TF_Service_dotNet/TouchFree/Configuration/ConfigFileUtils.cs

```diff
@@ -25,7 +25,7 @@ public static string ConfigFileDirectory
             }
         }
 
-        static readonly string DefaultConfigDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), "Ultraleap\\TouchFree\\Configuration\\");
+        static readonly string DefaultConfigDirectory = RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? Path.GetFullPath("/storage/sd/ultraleap/touchfree/configuration") : Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), "Ultraleap\\TouchFree\\Configuration\\");
```

Comment:
This line is quite long, might be nicer if we split the conditions onto separate lines but not essential

Reply:
It was originally multiple lines that I recombined just in case it was the cause of the syntax error being noticed when publishing (as opposed to building). Will revert that back shortly 👍